esiclient/tests/unit/v1/test_node_network.py

```diff
@@ -469,6 +473,22 @@ def test_take_action_no_port_or_network_or_trunk_exception(self):
             'ERROR: You must specify either network, port, or trunk',
             self.cmd.take_action, parsed_args)
 
+    def test_take_action_invalid_network_exception(self):
+        self.app.client_manager.baremetal.node.get.\
+            return_value = self.node
+        self.app.client_manager.baremetal.port.list.\
+            return_value = [self.port1]
+
```

Comment:
Oh, you want to mock the call to `find_network` as well to have it explicitly return `None`.